hela/dataset.py

```diff
@@ -0,0 +1,83 @@
+import os
+import json
+
+import numpy as np
+
+__all__ = ["Dataset", "load_dataset", "load_data_file"]
+
+
+class Dataset(object):
```

Comment:
The explicit class definition is only meant to produce more readable sphinx documentation, which gets built from the docstrings – this will be evident once the PR is merged and we get ReadTheDocs running. 👍